flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/runtime/operators/deduplicate/ProcTimeDeduplicateKeepLastRowFunctionTest.java

```diff
@@ -92,14 +102,31 @@ public void testWithGenerateUpdateBefore() throws Exception {
         testHarness.open();
         testHarness.processElement(insertRecord("book", 1L, 12));
         testHarness.processElement(insertRecord("book", 2L, 11));
-        testHarness.processElement(insertRecord("book", 1L, 13));
+        testHarness.processElement(insertRecord("book", 1L, 12));
         testHarness.close();
 
         // Keep LastRow in deduplicate may send UPDATE_BEFORE
         List<Object> expectedOutput = new ArrayList<>();
         expectedOutput.add(insertRecord("book", 1L, 12));
         expectedOutput.add(updateBeforeRecord("book", 1L, 12));
-        expectedOutput.add(updateAfterRecord("book", 1L, 13));
+        expectedOutput.add(updateAfterRecord("book", 1L, 12));
+        expectedOutput.add(insertRecord("book", 2L, 11));
+        assertor.assertOutputEqualsSorted("output wrong.", expectedOutput, testHarness.getOutput());
+    }
+
+    @Test
+    public void testDeduplicateWithGenerateUpdateBefore() throws Exception {
```

Comment:
Please use a more suitable method name for this test, this method name sounds same with the above one `testWithGenerateUpdateBefore`. What do you think about `testWithStateTtlDisabled`.

Reply:
ok，I think `testWithStateTtlDisabled` is better. I will change the method name

Reply:
done with commit https://github.com/apache/flink/pull/14863/commits/748ad82745d9d493922150fe007136e125a50209